packages/block-renderer/src/hooks/use-pattern-min-height-vh.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+
+const usePatternMinHeightVh = ( html = '', viewportHeight: number | undefined = 0 ) => {
+	return useMemo( () => {
+		return html.replace( /min-height:\s?(?<value>\d+)vh;?/g, ( match, value ) => {
+			if ( viewportHeight ) {
```

Comment:
With this change, I guess we can always pass the `viewportHeight` to the `PatternRenderer`, and then remove the logic related to the min-height for 100vh (e.g.: `isMinHeight100vh``heightFor100vh`) since the pattern won't have the vh anymore. WDYT?

Reply:
I tested patterns with 100vh and they still require that logic. That's why I haven't removed it. 

<img width="1472" alt="Screenshot 2566-08-31 at 16 56 39" src="https://github.com/Automattic/wp-calypso/assets/1881481/41b91d7a-c63b-448f-9c44-7c31fb1a0441">

Reply:
I agree that we could try to clean the logic for 100vh, let's do that later in another PR as is not a priority for now.

Reply:
Sure. Let's create a follow-up issue 🙂

Reply:
Created https://github.com/Automattic/wp-calypso/issues/81269